frontend/src/layout/navigation-3000/sidepanel/panels/activation/activationLogic.ts

```diff
@@ -282,6 +280,17 @@ export const activationLogic = kea<activationLogicType>([
                                 skipped: skippedTasks.includes(ActivationTasks.InstallFirstApp),
                             })
                             break
+                        case ActivationTasks.SetupReverseProxy:
```

Comment:
```suggestion
                        case ActivationTasks.SetUpReverseProxy:
```

and all the other instances 😬 😅